integration-tests/bot/chainbot.py

```diff
@@ -173,17 +173,17 @@ def extract_enckey(s):
 def app_state_cfg(cfg):
     return {
         "distribution": gen_distribution(cfg),
-        "unbonding_period": 60,
+        "unbonding_period": 20,
```

Comment:
These configurations can be customized in the `cluster.json`, for example: `integration-tests/multinode/jail_cluster.json`.

Reply:
Yeah, I know, I did that initially, but then I realized the "default" one wasn't valid anymore and every customization had to fix it, so I thought it'd be better just to fix the default one